doc/en/user/source/extensions/importer/rest_reference.rst

```diff
@@ -913,6 +913,28 @@ Remaps a certain field to a given target data type
      - The "target" field type, as a fully qualified Java class name
 
 
+AttributeComputeTransform
+"""""""""""""""""""""""""
+
+Computes a new field based on an expression fed the other field values
```

Comment:
Looks like there is a mistake here ``expression fed the``.

Reply:
"fed" past of "feed", the expression is given the other field values.
I'll rewrite to make it clearer